src/native-common/GestureView.tsx

```diff
@@ -298,7 +298,8 @@ export abstract class GestureView extends ViewBase<Types.GestureViewProps, {}> {
         }
 
         // Has the user started to pan?
-        const panThreshold = this.props.panPixelThreshold > 0 ? this.props.panPixelThreshold : _panPixelThreshold;
+        const panThreshold = (this.props.panPixelThreshold && this.props.panPixelThreshold > 0) ?
```

Comment:
The logic happens to work OK if the value is defined and equal to zero, but it would be better to make the check for undefined explicit.